libs/platform/form/combobox/combobox/combobox.component.ts

```diff
@@ -213,10 +213,15 @@ export class ComboboxComponent extends BaseCombobox implements ComboboxInterface
 
     /** @hidden Define is selected item selected */
     isSelectedOptionItem(selectedItem: any): boolean {
-        return (
-            (this.lookupKey && this.lookupValue(this._selectedElement) === this.lookupValue(selectedItem)) ||
-            this.displayValue(this._selectedElement) === this.displayValue(selectedItem)
-        );
+        if (!!this._selectedElement && !!selectedItem) {
+            if (this.lookupKey) {
+                return this.lookupValue(this._selectedElement) === this.lookupValue(selectedItem);
+            } else {
+                return selectedItem === this._selectedElement;
+            }
+        } else {
+            return false;
+        }
```

Comment:
```suggestion
        if (!!this._selectedElement && !!selectedItem) {
            if (this.lookupKey) {
                return this.lookupValue(this._selectedElement) === this.lookupValue(selectedItem);
            }
            return selectedItem === this._selectedElement;
        }
        return false;
```
I'd say, there is no need for `else`